### ISSUE TITLE
Hide other surah's while reading one surah

### ISSUE THREAD

**Issue by Gamer750** [Waqar144/quran_memorization_helper#5]

**Comment by Gamer750** [comment]
Any update

**Comment by Waqar144** [comment]
No, because this feature is impractical so I am not sure if this will ever get implemented. Vast majority of the surahs are bigger than 1 page.

**Comment by Gamer750** [comment]
> No, because this feature is impractical so I am not sure if this will ever get implemented. Vast majority of the surahs are bigger than 1 page.

Other apps have this feature . I am not trying to be rude or something

**Comment by Waqar144** [comment]
Merging this with #19
